cockatrice/src/window_main.cpp

```diff
@@ -334,10 +337,10 @@ void MainWindow::loginError(Response::ResponseCode r, QString reasonStr, quint32
     switch (r) {
         case Response::RespClientUpdateRequired: {
             QString formattedMissingFeatures;
-            formattedMissingFeatures = "Missing Features: ";
+            formattedMissingFeatures = tr("Missing Features:") + QString(" ");
             for (int i = 0; i < missingFeatures.size(); ++i)
                 formattedMissingFeatures.append(QString("\n     %1").arg(QChar(0x2022)) + " " + missingFeatures.value(i)   );
-            formattedMissingFeatures.append("\nTo update your client, go to Help -> Check for Updates.");
+            formattedMissingFeatures.append(QString("\n") + tr("To update your client, go to \"Help -> Check for Client Updates\"."));
```

Comment:
Because it should mirror the exact phrasing of our menu item firstly.

We don't use this style of formatting in a uniform way at the moment... but as any other program (Word, Photoshop...) it's the way you label things in menus, names for windows, headlines etc.
It's called "title case" or "title capitalization", every newspaper uses it as well for captions for example.